prediction_market_agent/agents/prophet_agent/deploy.py

```diff
@@ -58,6 +69,44 @@ def load(self) -> None:
         )
 
 
+class DeployablePredictionProphetGPT4oAgentNewMarketTrader(
+    DeployablePredictionProphetGPT4oAgent
+):
+    """
+    This agent trades on new markets, then re-evaluates the market over its
+    lifetime, if it observes that news has been published about the market since
+    its last trade.
+    """
+
+    bet_on_n_markets_per_run = 20
+    trade_on_markets_created_after = DatetimeUTC(2024, 10, 31, 0)
```

Comment:
Why is this necessary for this agent?

Reply:
Hmm well it's not necessary, but it fits the description of betting on 'new markets, and re-evaluates them over their lifetime'. An alternative solution would be to get the agent's deployment `start_time` at runtime, but it's still not great because it assumes that env var exists

Reply:
That makes sense! Thanks